dissect/target/plugins/apps/browser/brave.py

```diff
@@ -66,3 +71,14 @@ def downloads(self) -> Iterator[BrowserDownloadRecord]:
     def extensions(self) -> Iterator[BrowserExtensionRecord]:
         """Return browser extension records for Brave."""
         yield from super().extensions("brave")
+
+    @export(record=BrowserPasswordRecord)
+    @arg(
+        "--passwords",
+        type=str,
+        default="",
+        help="Supply plaintext Windows passwords or SHA1 hashes in comma delimited fashion.",
+    )
```

Comment:
Perhaps it's a good idea to extend the [keychain](https://github.com/fox-it/dissect.target/blob/main/dissect/target/helpers/keychain.py) for this kind of functionality.

Reply:
Perhaps a little bit of both is necessary, since for Chromium you'll need user profile passwords (for which I think it might be better to implement it in DPAPI directly) but for Firefox you'll need the master password (which is obviously Firefox specific).

Quickly looking at the current specifications of the keychain, I don't expect too many changes to the format. Provider and key type are still relevant, identifier could be a SID or Firefox profile ID. Perhaps something that will need some improvement is the way we can provide passphrases on the CLI. I've been wanting an extension of that anyway, with perhaps some inline way of specifying the identifier. 

Feel free to make some suggestions, we can probably riff of off that.